app/views/foreman_kernel_care/job_templates/update_kernel.erb

```diff
@@ -3,13 +3,19 @@ kind: job_template
 name: Update kernel
 model: JobTemplate
 job_category: Commands
-description_format: "Run yum update kernel"
+description_format: "Update kernel"
 provider_type: SSH
 feature: update_kernel
 %>
-<%
-  unless @host.operatingsystem.family == 'Redhat'
-    render_error(N_('Unsupported operating system for this host.'))
-  end
-%>
-yum update -y kernel*
+
+<%= snippet 'pkg_manager' %>
+
+<% if @host.operatingsystem.family == 'Redhat' -%>
+KERNEL_NAME="kernel*"
+<% elsif @host.operatingsystem.family == 'Debian' -%>
+KERNEL_NAME="linux-image-generic"
+<% else -%>
+<%= render_error(N_('Unsupported operating system for this host.')) %>
+<% end %>
+
+$PKG_MANAGER_UPGRADE $KERNEL_NAME
```

Comment:
currently, this template does install the kernel from OS vendor but it doesn't call `kcarectl --update`. Wouldn't it make sense, to run this command, too? Or is this done somehow different?

Reply:
I think you right. I will add `kcarectl --update` to this job